frontend/src/scenes/saved-insights/SavedInsights.tsx

```diff
@@ -0,0 +1,151 @@
+    const { hasDashboardCollaboration } = useValues(userLogic)
+    const [displayedColumns, setDisplayedColumns] = useState([] as ColumnType<DashboardItemType>[])
+
+    useEffect(() => {
+        loadInsights()
+    }, [])
+
+    useEffect(() => {
+        if (!hasDashboardCollaboration) {
+            setDisplayedColumns(columns.filter((col) => !col.dataIndex || !['tags'].includes(col.dataIndex.toString())))
+        } else {
+            setDisplayedColumns(columns)
+        }
+    }, [hasDashboardCollaboration])
+
+    const columns = [
+        {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+            render: function renderName(
+                name: string,
+                {
+                    short_id,
+                    id,
+                    description,
+                    favorited,
+                }: { short_id: string; id: number; description?: string; favorited?: boolean }
+            ) {
+                return (
+                    <Col>
+                        <Row>
+                            <Link to={`/i/${short_id}`} style={{ marginRight: 12 }}>
+                                <strong>{name || `Insight #${id}`}</strong>
+                            </Link>
+                            <div
+                                style={{ cursor: 'pointer', width: 'fit-content' }}
+                                onClick={() => updateFavoritedInsight({ id, favorited: !favorited })}
+                            >
+                                {favorited ? (
+                                    <StarFilled className="text-warning" />
+                                ) : (
+                                    <StarOutlined className="star-outlined" />
+                                )}
+                            </div>
+                        </Row>
+                        {hasDashboardCollaboration && (
+                            <div className="text-muted-alt">{description || 'No description provided'}</div>
+                        )}
+                    </Col>
+                )
+            },
+        },
+        {
+            title: 'Tags',
+            dataIndex: 'tags',
+            key: 'tags',
+            render: function renderTags(tags: string[]) {
+                return <ObjectTags tags={tags} staticOnly />
+            },
+        },
+        {
+            title: 'Last modified',
+            dataIndex: 'updated_at',
+            key: 'updated_at',
+            render: function renderLastModified(updated_at: string) {
+                return <span>{humanFriendlyDetailedTime(updated_at)}</span>
+            },
+            sorter: (a: DashboardItemType, b: DashboardItemType) =>
+                new Date(a.updated_at) > new Date(b.updated_at) ? 1 : -1,
+        },
+        createdByColumn(insights.results) as ColumnType<DashboardItemType>,
```

Comment:
Tangential: We could add the profile picture to this column for the created by users